README.md

```diff
@@ -30,3 +30,43 @@ to initalise the database, after which you can start hacking :)
 
 Run `make test` (or `poetry run pytest`) to run the tests and `make server` (or
 `poetry run ./manage.py runserver`) to run the development server.
+
+## Testing
+
+To run all of the test, a instance of Celery must be running. From `resotool`,
+this can either be achieved by running `make celery-run` for a simple Celery
+worker or `make celery-start` for a detached celery worker, which can be
+stopped with `make celery-stop` after running the tests. If no Celery instance
+is running, all tests requiring Celery will be skipped.
+
+### Celery mails
+
+Since the resotool uses a separate back end to send mails using Celery and
+Django replaces the set email back end with the `locmem` back end, there is a
```

Comment:
back end -> backend

Reply:
I fixed the other typos, but with this I'm not sure. Everywhere I look, it's either spelled "back end" or there doesn't seem to be a real difference between "backend" and "back end".

Reply:
Actually, I just learned that most dictionaries refer to this as "back end" and Wikipedia knows "back end", "back-end" and "backend". TIL.